packages/react-native-bridge/ios/Gutenberg.swift

```diff
@@ -184,19 +181,32 @@ public class Gutenberg: NSObject {
         bridgeModule.sendEventIfNeeded(.setFocusOnTitle, body: nil)
     }
 
-    public func updateTheme(_ editorTheme: GutenbergEditorTheme?) {
+    public func updateEditorSettings(_ editorSettings: GutenbergEditorSettings?) {
+        let settingsUpdates = properties(from: editorSettings)
+        sendEvent(.updateEditorSettings, body: settingsUpdates)
+    }
+
+    private func properties(from editorSettings: GutenbergEditorSettings?) -> [String : Any] {
+        var settingsUpdates = [String : Any]()
+        settingsUpdates["isFSETheme"] = editorSettings?.isFSETheme ?? false
```

Comment:
@antonis @geriux I pushed the iOS Changes in bf75be1. As part of that, I included `isFSETheme` which maps to `__unstableEnableFullSiteEditingBlocks` which may not be critical yet but I included it in case it helps identify if `colors` and `gradients` are blank because the theme wants the default values or if it's blank because we're using an FSE theme.